pages/api/v1/teams/[id]/roster/stats.ts

```diff
@@ -8,6 +8,9 @@ const cors = Cors({
   methods: ['GET', 'HEAD'],
 });
 
+const seasonTypes = ['Pre-Season', 'Regular Season', 'Playoffs'];
+export type SeasonType = typeof seasonTypes[number];
```

Comment:
What is this exported for? This is saying that seasonType is of type "String[]" when it should be "String". I don't see a use for these lines so we can remove them.

```suggestion
```